linux_os/guide/services/ftp/ftp_configure_vsftpd/ftp_present_banner/rule.yml

```
@@ -44,3 +44,5 @@ ocil: |-
     <pre>$ sudo grep "banner_file" /etc/vsftpd/vsftpd.conf
     {{% endif %}}
     banner_file=/etc/issue</pre>
+
+{{{ warning_rule_deprecated_cause("SLES 12 STIG Revision v2R10", release='0.1.69') | indent(4) }}}
```

I meant something like this:
```suggestion
{{%- if product in ["sle12"] %}}
{{{ warning_rule_deprecated_cause("SLES 12 STIG Revision v2R10", release='0.1.69') | indent(4) }}}
{{%- endif %}}
```
I am not sure if this rule is also deprecated for other products in `prodtype`, so I would keep the warning only for the products we are sure now.